.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,50 @@
+{
+  "name": "Python 3",
+  "build": {
+    "dockerfile": "./Dockerfile"
+  },
+  "features": {
+    "ghcr.io/deepspacecartel/devcontainers-features/apt:2": {
+      "packages": "shellcheck bash-completion cmake libcairo2-dev pkg-config python3-dev ffmpeg libsm6 libxext6 openscad pypy3 pypy3-dev graphviz pipx time" // shfmt
+    },
+    "ghcr.io/deepspacecartel/devcontainers-features/pip:2": {
+      "packages": "pre-commit poetry",
```

Comment:
- [ ] `pre-commit` should activate itself automatically.
- [ ] `poetry` should be explained in `./docs`.